homework-15

KhrysCS opened the pull request:
Completed homework-15

Updated LoginTests.java to change the URL from "https://qa.koel.app/" to "https://testpro.io/".

KhrysCS committed:
Update LoginTests.java for homework-15

Changed the URL from "https://qa.koel.app/" to "https://testpro.io/".

KhrysCS committed:
Merge pull request #1 from KhrysCS/homework-15

Update LoginTests.java for homework-15

khanhtestpro commented:
//Comment from HW team,
Hello,
Your homework has been reviewed.
Great job on creating your Homework 15 and change the URL from "https://qa.koel.app/" to "https://testpro.io/".
https://share.zight.com/eDupWl2J

You are providing incorrect PR link.

Here is an example of a correct pull request. 

Plz resubmit with correct link.

Example (DO NOT USE):https://github.com/[your-github-username)/A40-QA-Automation-Framework/pull/1

Best wishes from the HW team (Khanh, khanh@testpro.io)

KhrysCS committed:
Adding class Homework16

Added a new java file containing the class Homework16 and the corresponding code for the test case registrationNavigation.

Added Homework16  to TestNG.xml.

KhrysCS committed:
Merge pull request #2 from KhrysCS/homework-16

Adding class Homework16

KhrysCS committed:
Adding practice code from lesson M6-L6

Adding practice code from the slide from lesson M6-L6 and testcase from homework16.

KhrysCS committed:
Refactored for lesson 16

Refactored all the code like in lesson 16.

KhrysCS committed:
Merge pull request #3 from KhrysCS/M6-L17

Refactored for lesson 16

KhrysCS committed:
Added class Homework17

Added new class for Homework17

KhrysCS committed:
Merge pull request #4 from KhrysCS/homework-17

Added class Homework17

KhrysCS committed:
Homework18

Added class Homework18 and SmokeTest.xml.
Also modified TestNG.xml to run SmokeTest,.xml

KhrysCS committed:
Merge pull request #5 from KhrysCS/homework-18

Homework18

KhrysCS committed:
edited build,gradle

Edited build.gradle file to run TestNG.xml

KhrysCS committed:
M6-L19 code changes

M6-L19 code changes for lesson.

KhrysCS committed:
Merge pull request #6 from KhrysCS/M6-L19

M6-L19 code changes

KhrysCS committed:
Homework M6-L19

Homework M6-L19 Framework: Additional Annotations & Data Providers
Added class Homework19.java which has all the functionality required for the test case deletePlaylist()
Added class name Homework19 to TestNG file
Commented class LoginTests in TestNG file

KhrysCS committed:
Merge pull request #7 from KhrysCS/homework-19

Homework M6-L19

KhrysCS committed:
Homework M7-L20 Selenium: Waits

Homework M7-L20 Selenium: Waits
Added new class Homework20 to repeat test case deletePlaylist but this time using waits instead of thread sleep.
Modified BaseTest.java and LoginTests.java to remove all thread sleeps and replace with waits.

KhrysCS committed:
Merge pull request #8 from KhrysCS/homework-20

Homework M7-L20 Selenium: Waits

KhrysCS committed:
M8-L21 Selenium: Web Elements and Action class Slides Examples

M8-L21 Selenium: Web Elements and Action class Slides Examples
Added classes:
ContextClick
CountPlaylistSongs
DoubleClick
Hover

KhrysCS committed:
Merge pull request #9 from KhrysCS/M8-L21

M8-L21 Selenium: Web Elements and Action class Slides Examples

KhrysCS committed:
Homework M8-L21 Selenium: Web Elements and Action class

Homework M8-L21 Selenium: Web Elements and Action class

Added class Homework21.java that contains the required  functionality for Test case renamePlaylist using Action Class methods.

KhrysCS committed:
Merge pull request #10 from KhrysCS/homework-21

Homework M8-L21 Selenium: Web Elements and Action class

KhrysCS committed:
M9-L22 Selenium: Page Object Model (POM) Slide Examples

M9-L22 Selenium: Page Object Model (POM) Slide Examples
Added package pages
Added classes BasePage, LoginPage and HomePage
Modified LoginTests to use new classes

KhrysCS committed:
Merge pull request #11 from KhrysCS/M9-L22

M9-L22 Selenium: Page Object Model (POM) Slide Examples

KhrysCS committed:
Homework M9-L22 Selenium: Page Object Model (POM)

Homework M9-L22 Selenium: Page Object Model (POM)
Modified LoginTests method for  Test case Login valid email empty password.
Method now uses methods and objects from LoginPage

KhrysCS committed:
Merge pull request #12 from KhrysCS/homework-22

Homework M9-L22 Selenium: Page Object Model (POM)

KhrysCS committed:
M9-L23 Selenium: Page Factory

M9-L23 Selenium: Page Factory

Modified BasePage,LoginPage and LoginTests to use Page factory and fluent interfaces.

KhrysCS committed:
Merge pull request #13 from KhrysCS/M9-L23

M9-L23 Selenium: Page Factory

KhrysCS committed:
Homework M9-L23 Selenium: Page Factory

Homework M9-L23 Selenium: Page Factory
Modified LoginTest Invalid Email Valid Password Test case to use Page Factory and Fluent Interface.
Modified HomePage to use Page Factory

KhrysCS committed:
Merge pull request #14 from KhrysCS/homework-23

Homework M9-L23 Selenium: Page Factory

KhrysCS committed:
Homework M9-L24 Selenium: Grid

Homework M9-L24 Selenium: Grid
Changes made in build.gradle, BaseTest and TestNG to be able to test from any browser and to use Selenium Grid

KhrysCS committed:
Merge pull request #15 from KhrysCS/homework-24

Homework M9-L24 Selenium: Grid
